components/form/Toggle.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import classNames from 'classnames';
+
+const Toggle = ({ containerClassName, label, name, value, toggleClassName }) => {
```

Comment:
Since we are in the toggle component, we can use `className` instead of `toggleClassName`. 

```suggestion
const Toggle = ({ containerClassName, label, name, value, className }) => {
```